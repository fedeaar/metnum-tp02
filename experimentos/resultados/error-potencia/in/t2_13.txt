6.361561000000000000e+06 -2.795999000000000000e+06 -4.211400000000000000e+05 6.256500000000000000e+04 -5.682710000000000000e+05 4.695380000000000000e+05 6.864010000000000000e+05 -1.560551000000000000e+06 -2.881793000000000000e+06 1.872780000000000000e+05 8.923790000000000000e+05 1.319961000000000000e+06 -8.026440000000000000e+05 -1.955560000000000000e+05 5.890050000000000000e+05 -2.247690000000000000e+06 -3.445380000000000000e+05 1.719943000000000000e+06 1.975681000000000000e+06 3.738260000000000000e+05
-2.795999000000000000e+06 5.664264000000000000e+06 -1.314404000000000000e+06 2.567926000000000000e+06 3.071270000000000000e+06 7.311790000000000000e+05 -8.903900000000000000e+05 -1.700000000000000000e+03 1.006981000000000000e+06 -2.820993000000000000e+06 -2.548369000000000000e+06 -9.240750000000000000e+05 6.036140000000000000e+05 5.987220000000000000e+05 -3.225180000000000000e+05 1.599657000000000000e+06 1.066296000000000000e+06 -9.703570000000000000e+05 -8.530700000000000000e+04 2.043000000000000000e+03
-4.211400000000000000e+05 -1.314404000000000000e+06 4.866335000000000000e+06 -1.137370000000000000e+06 -1.176669000000000000e+06 5.570360000000000000e+05 1.273997000000000000e+06 2.632510000000000000e+05 -4.676590000000000000e+05 2.247806000000000000e+06 5.044660000000000000e+05 -1.686931000000000000e+06 -1.714120000000000000e+05 -1.509057000000000000e+06 -6.519850000000000000e+05 -1.329963000000000000e+06 -2.457738000000000000e+06 2.957800000000000000e+05 8.603100000000000000e+04 -2.125973000000000000e+06
6.256500000000000000e+04 2.567926000000000000e+06 -1.137370000000000000e+06 6.463563000000000000e+06 1.985792000000000000e+06 1.309719000000000000e+06 -1.750768000000000000e+06 -1.314824000000000000e+06 8.996560000000000000e+05 -3.050221000000000000e+06 -1.119868000000000000e+06 9.157100000000000000e+04 1.163372000000000000e+06 5.488070000000000000e+05 -9.426610000000000000e+05 1.315919000000000000e+06 9.715390000000000000e+05 2.991450000000000000e+05 1.563038000000000000e+06 7.862970000000000000e+05
-5.682710000000000000e+05 3.071270000000000000e+06 -1.176669000000000000e+06 1.985792000000000000e+06 7.223847000000000000e+06 -3.102592000000000000e+06 -1.727197000000000000e+06 -1.448183000000000000e+06 -7.098990000000000000e+05 -1.222645000000000000e+06 -3.005359000000000000e+06 1.308161000000000000e+06 -5.516320000000000000e+05 1.805494000000000000e+06 -6.139900000000000000e+04 2.449626000000000000e+06 -7.431240000000000000e+05 -4.502027000000000000e+06 -4.732820000000000000e+05 -1.527170000000000000e+05
4.695380000000000000e+05 7.311790000000000000e+05 5.570360000000000000e+05 1.309719000000000000e+06 -3.102592000000000000e+06 7.646207000000000000e+06 1.730401000000000000e+06 -3.315840000000000000e+05 1.150435000000000000e+06 -1.555015000000000000e+06 1.386598000000000000e+06 -1.713819000000000000e+06 -1.164235000000000000e+06 -6.804230000000000000e+05 -1.947640000000000000e+05 -9.475780000000000000e+05 2.902110000000000000e+05 4.266333000000000000e+06 2.272488000000000000e+06 6.279310000000000000e+05
6.864010000000000000e+05 -8.903900000000000000e+05 1.273997000000000000e+06 -1.750768000000000000e+06 -1.727197000000000000e+06 1.730401000000000000e+06 6.685241000000000000e+06 -2.699770000000000000e+05 -1.244388000000000000e+06 3.283750000000000000e+06 2.234914000000000000e+06 -3.122469000000000000e+06 -9.104730000000000000e+05 -3.153756000000000000e+06 -3.408727000000000000e+06 -1.415060000000000000e+06 -3.342080000000000000e+05 4.634670000000000000e+05 3.985826000000000000e+06 -2.913050000000000000e+06
-1.560551000000000000e+06 -1.700000000000000000e+03 2.632510000000000000e+05 -1.314824000000000000e+06 -1.448183000000000000e+06 -3.315840000000000000e+05 -2.699770000000000000e+05 7.355271000000000000e+06 1.081720000000000000e+05 -4.196710000000000000e+05 5.976340000000000000e+05 -2.906458000000000000e+06 4.085680000000000000e+05 2.213600000000000000e+04 -2.127100000000000000e+05 6.885090000000000000e+05 1.341552000000000000e+06 -1.655924000000000000e+06 7.773290000000000000e+05 -2.402066000000000000e+06
-2.881793000000000000e+06 1.006981000000000000e+06 -4.676590000000000000e+05 8.996560000000000000e+05 -7.098990000000000000e+05 1.150435000000000000e+06 -1.244388000000000000e+06 1.081720000000000000e+05 6.323489000000000000e+06 -4.115680000000000000e+05 2.700725000000000000e+06 9.994920000000000000e+05 2.209397000000000000e+06 5.852480000000000000e+05 9.982300000000000000e+05 -1.935817000000000000e+06 -8.610870000000000000e+05 1.087870000000000000e+06 -2.585650000000000000e+05 2.321003000000000000e+06
1.872780000000000000e+05 -2.820993000000000000e+06 2.247806000000000000e+06 -3.050221000000000000e+06 -1.222645000000000000e+06 -1.555015000000000000e+06 3.283750000000000000e+06 -4.196710000000000000e+05 -4.115680000000000000e+05 7.557523000000000000e+06 6.397440000000000000e+05 -2.830600000000000000e+05 -4.354450000000000000e+05 -4.231746000000000000e+06 -8.983560000000000000e+05 -2.382596000000000000e+06 -1.160927000000000000e+06 -7.871630000000000000e+05 -1.804600000000000000e+05 7.444000000000000000e+04
8.923790000000000000e+05 -2.548369000000000000e+06 5.044660000000000000e+05 -1.119868000000000000e+06 -3.005359000000000000e+06 1.386598000000000000e+06 2.234914000000000000e+06 5.976340000000000000e+05 2.700725000000000000e+06 6.397440000000000000e+05 9.009504000000000000e+06 8.094580000000000000e+05 1.147364000000000000e+06 -2.542725000000000000e+06 -1.924023000000000000e+06 -5.713009000000000000e+06 -1.373899000000000000e+06 3.354773000000000000e+06 2.246093000000000000e+06 -1.235001000000000000e+06
1.319961000000000000e+06 -9.240750000000000000e+05 -1.686931000000000000e+06 9.157100000000000000e+04 1.308161000000000000e+06 -1.713819000000000000e+06 -3.122469000000000000e+06 -2.906458000000000000e+06 9.994920000000000000e+05 -2.830600000000000000e+05 8.094580000000000000e+05 5.933508000000000000e+06 1.088401000000000000e+06 9.548920000000000000e+05 2.715484000000000000e+06 -1.195242000000000000e+06 -5.110040000000000000e+05 1.726430000000000000e+05 -2.873990000000000000e+06 2.987136000000000000e+06
-8.026440000000000000e+05 6.036140000000000000e+05 -1.714120000000000000e+05 1.163372000000000000e+06 -5.516320000000000000e+05 -1.164235000000000000e+06 -9.104730000000000000e+05 4.085680000000000000e+05 2.209397000000000000e+06 -4.354450000000000000e+05 1.147364000000000000e+06 1.088401000000000000e+06 4.706660000000000000e+06 7.403330000000000000e+05 7.308760000000000000e+05 -2.894451000000000000e+06 8.133520000000000000e+05 2.400080000000000000e+05 -1.852415000000000000e+06 -1.525720000000000000e+05
-1.955560000000000000e+05 5.987220000000000000e+05 -1.509057000000000000e+06 5.488070000000000000e+05 1.805494000000000000e+06 -6.804230000000000000e+05 -3.153756000000000000e+06 2.213600000000000000e+04 5.852480000000000000e+05 -4.231746000000000000e+06 -2.542725000000000000e+06 9.548920000000000000e+05 7.403330000000000000e+05 6.231838000000000000e+06 2.510769000000000000e+06 3.111344000000000000e+06 3.269710000000000000e+05 -8.009450000000000000e+05 -2.161462000000000000e+06 4.960620000000000000e+05
5.890050000000000000e+05 -3.225180000000000000e+05 -6.519850000000000000e+05 -9.426610000000000000e+05 -6.139900000000000000e+04 -1.947640000000000000e+05 -3.408727000000000000e+06 -2.127100000000000000e+05 9.982300000000000000e+05 -8.983560000000000000e+05 -1.924023000000000000e+06 2.715484000000000000e+06 7.308760000000000000e+05 2.510769000000000000e+06 6.614127000000000000e+06 3.694680000000000000e+05 -7.203670000000000000e+05 5.844940000000000000e+05 -3.076818000000000000e+06 2.473735000000000000e+06
-2.247690000000000000e+06 1.599657000000000000e+06 -1.329963000000000000e+06 1.315919000000000000e+06 2.449626000000000000e+06 -9.475780000000000000e+05 -1.415060000000000000e+06 6.885090000000000000e+05 -1.935817000000000000e+06 -2.382596000000000000e+06 -5.713009000000000000e+06 -1.195242000000000000e+06 -2.894451000000000000e+06 3.111344000000000000e+06 3.694680000000000000e+05 8.237354000000000000e+06 1.881209000000000000e+06 -3.174221000000000000e+06 -7.619690000000000000e+05 -7.737500000000000000e+04
-3.445380000000000000e+05 1.066296000000000000e+06 -2.457738000000000000e+06 9.715390000000000000e+05 -7.431240000000000000e+05 2.902110000000000000e+05 -3.342080000000000000e+05 1.341552000000000000e+06 -8.610870000000000000e+05 -1.160927000000000000e+06 -1.373899000000000000e+06 -5.110040000000000000e+05 8.133520000000000000e+05 3.269710000000000000e+05 -7.203670000000000000e+05 1.881209000000000000e+06 6.089995000000000000e+06 -8.430000000000000000e+02 -2.408453000000000000e+06 4.145130000000000000e+05
1.719943000000000000e+06 -9.703570000000000000e+05 2.957800000000000000e+05 2.991450000000000000e+05 -4.502027000000000000e+06 4.266333000000000000e+06 4.634670000000000000e+05 -1.655924000000000000e+06 1.087870000000000000e+06 -7.871630000000000000e+05 3.354773000000000000e+06 1.726430000000000000e+05 2.400080000000000000e+05 -8.009450000000000000e+05 5.844940000000000000e+05 -3.174221000000000000e+06 -8.430000000000000000e+02 6.616355000000000000e+06 -4.611940000000000000e+05 1.288031000000000000e+06
1.975681000000000000e+06 -8.530700000000000000e+04 8.603100000000000000e+04 1.563038000000000000e+06 -4.732820000000000000e+05 2.272488000000000000e+06 3.985826000000000000e+06 7.773290000000000000e+05 -2.585650000000000000e+05 -1.804600000000000000e+05 2.246093000000000000e+06 -2.873990000000000000e+06 -1.852415000000000000e+06 -2.161462000000000000e+06 -3.076818000000000000e+06 -7.619690000000000000e+05 -2.408453000000000000e+06 -4.611940000000000000e+05 9.187719000000000000e+06 -1.669276000000000000e+06
3.738260000000000000e+05 2.043000000000000000e+03 -2.125973000000000000e+06 7.862970000000000000e+05 -1.527170000000000000e+05 6.279310000000000000e+05 -2.913050000000000000e+06 -2.402066000000000000e+06 2.321003000000000000e+06 7.444000000000000000e+04 -1.235001000000000000e+06 2.987136000000000000e+06 -1.525720000000000000e+05 4.960620000000000000e+05 2.473735000000000000e+06 -7.737500000000000000e+04 4.145130000000000000e+05 1.288031000000000000e+06 -1.669276000000000000e+06 5.666437000000000000e+06
