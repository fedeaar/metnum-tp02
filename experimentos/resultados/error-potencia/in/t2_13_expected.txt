2.848178336176727340e+07
2.020290152374326810e+07
1.589963781881620362e+07
1.253339828828984499e+07
1.166859223126678728e+07
9.018885595489289612e+06
8.321774221822749823e+06
6.484799198764726520e+06
5.197576435991061851e+06
4.146419804682540707e+06
3.817332447390955873e+06
2.406338674022778403e+06
2.216821356175820343e+06
1.606122557108947309e+06
1.035162945890900213e+06
6.724313123784917407e+05
3.685505637756654178e+05
3.167725665849327925e+05
4.014162789711368532e+04
5.355468140716807284e+03
